gef-ui/server/controllers/application-details/index.js

```diff
@@ -249,9 +254,12 @@ const applicationDetails = async (req, res, next) => {
       params.errors = req.errors;
     }
 
-    if (req.success) {
-      params.success = req.success;
-    }
+    // Retrieve success message from req.flash
+    const [successMessage] = req.flash('success') || ['Facility is updated'];
```

Comment:
Where are we setting the req.flash that we can pull it out here?

Reply:
Ah I've seen that it's populated before a redirect. Is this message now making it so every time we load this page we're having a successMessage even if we didn't get to this page from a redirect? Sorry if I'm misunderstanding

Reply:
Hm, concur that OR part (`|| ['Facility is updated']`) shouldn't be required.

Reply:
Removed (|| ['Facility is updated'])